sentry-spring-boot-jakarta/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1 @@
+org.springframework.context.ApplicationListener=io.sentry.spring.boot.jakarta.SentrySpringVersionChecker
```

Comment:
`h` does this still work with Spring `3.3.0-M2`?

Reply:
Tested with `3.3.0-M2` and is working as expected. Seems like this change was only targeting `auto-configuration`.
Also the current documentation for `3.3.0-M2` mentions `spring.factories` as the place to register listeners: https://docs.spring.io/spring-boot/docs/3.3.0-M2/reference/htmlsingle/#features.spring-application.application-events-and-listeners.